include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,553 @@
+
+    /* ImageViewMinLodFeaturesEXT *//* VK_EXT_image_view_min_lod */
+    //VkBool32           minLod;
+
+    /* IndexTypeUint8FeaturesEXT *//* VK_EXT_index_type_uint8 */
+    //VkBool32           indexTypeUint8;
+    
+    /* InlineUniformBlockFeaturesEXT *//* VK_EXT_inline_uniform_block *//* MOVED TO Vulkan 1.3 Core */
+
+    /* LineRasterizationFeaturesEXT *//* VK_EXT_line_rasterization */
+    //VkBool32           rectangularLines;
+    //VkBool32           bresenhamLines;
+    //VkBool32           smoothLines;
+    //VkBool32           stippledRectangularLines;
+    //VkBool32           stippledBresenhamLines;
+    //VkBool32           stippledSmoothLines;
+
+    /* MemoryPriorityFeaturesEXT *//* VK_EXT_memory_priority */
+    //VkBool32           memoryPriority;
+
+    /* MultiDrawFeaturesEXT *//* VK_EXT_multi_draw */
+    //VkBool32           multiDraw;
+
+    /* PageableDeviceLocalMemoryFeaturesEXT *//* VK_EXT_pageable_device_local_memory */
+    //VkBool32           pageableDeviceLocalMemory;
+
+    /* PipelineCreationCacheControlFeaturesEXT *//* VK_EXT_pipeline_creation_cache_control *//* MOVED TO Vulkan 1.3 Core */
+
+    /* PrimitivesGeneratedQueryFeaturesEXT *//* VK_EXT_primitives_generated_query */
+    //VkBool32           primitivesGeneratedQuery;
+    //VkBool32           primitivesGeneratedQueryWithRasterizerDiscard;
+    //VkBool32           primitivesGeneratedQueryWithNonZeroStreams;
+
+    /* PrimitiveTopologyListRestartFeaturesEXT *//* VK_EXT_primitive_topology_list_restart */
+    //VkBool32           primitiveTopologyListRestart;
+    //VkBool32           primitiveTopologyPatchListRestart;
+
+    /* PrivateDataFeatures *//* VK_EXT_private_data *//* MOVED TO Vulkan 1.3 Core */
+
+    /* ProvokingVertexFeaturesEXT *//* VK_EXT_provoking_vertex */
+    //VkBool32           provokingVertexLast;
+    //VkBool32           transformFeedbackPreservesProvokingVertex;
+
+    /* Robustness2FeaturesEXT *//* VK_EXT_robustness2 */
+    //VkBool32           robustBufferAccess2;
+    //VkBool32           robustImageAccess2;
+    //VkBool32           nullDescriptor;
+    
+    /* ScalarBlockLayoutFeaturesEXT *//* VK_EXT_scalar_block_layout *//* MOVED TO Vulkan 1.2 Core */
+    
+    /* ShaderAtomicFloatFeaturesEXT *//* VK_EXT_shader_atomic_float */
+    //VkBool32           shaderBufferFloat32Atomics;
+    //VkBool32           shaderBufferFloat32AtomicAdd;
+    //VkBool32           shaderBufferFloat64Atomics;
+    //VkBool32           shaderBufferFloat64AtomicAdd;
+    //VkBool32           shaderSharedFloat32Atomics;
+    //VkBool32           shaderSharedFloat32AtomicAdd;
+    //VkBool32           shaderSharedFloat64Atomics;
+    //VkBool32           shaderSharedFloat64AtomicAdd;
+    //VkBool32           shaderImageFloat32Atomics;
+    //VkBool32           shaderImageFloat32AtomicAdd;
+    //VkBool32           sparseImageFloat32Atomics;
+    //VkBool32           sparseImageFloat32AtomicAdd;
```

Comment:
```cpp
if (NV_shader_atomic_float)
{
    shaderBufferFloat32Atomics = true;
    shaderBufferFloat32AtomicsAdd = true;
    shaderSharedFloat32Atomics = true;
    shaderSharedFloat32AtomicAdd = true;
    shaderSharedFloat32Atomics= true;
    shaderImageFloat32Atomics = true;
    shaderImageFloat32AtomicsAdd = true;
    // TODO: verify, not sure about those
    sparseImageFloat32Atomics = true;
    sparseImageFloat32AtomicAdd = true;
}
else if (INTEL_shader_atomic_float_minmax)
{
    shaderBufferFloat32Atomics = true;
    shaderSharedFloat32Atomics= true;
}

if (NV_shader_atomic_float64)
{
    shaderBufferFloat64Atomics = true;
    shaderBufferFloat64AtomicsAdd = true;
    shaderSharedFloat64Atomics = true;
    shaderSharedFloat64AtomicsAdd = true;
}
```

Reply:
unlike other "shader-only" extensions lets not enable by default because affects format promotion and validation